XamarinCommunityToolkit/Effects/SafeAreaEffectRouter.ios.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Linq;
+using UIKit;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.iOS;
+using XamarinCommunityToolkit.Effects;
+using Effects = XamarinCommunityToolkit.iOS.Effects;
+
+[assembly: ResolutionGroupName(nameof(XamarinCommunityToolkit))]
+[assembly: ExportEffect(typeof(Effects.SafeAreaEffectRouter), nameof(SafeAreaEffectRouter))]
+namespace XamarinCommunityToolkit.iOS.Effects
+{
+    public class SafeAreaEffectRouter : PlatformEffect
+    {
+        Thickness initialMargin;
+
+        protected override void OnAttached()
+        {
+            if (!(Element is View element))
+                return;
+
+            if (!UIDevice.CurrentDevice.CheckSystemVersion(11, 0) || !UIApplication.SharedApplication.Windows.Any())
```

Comment:
Since we're using Forms anyway. Isn't there a Forms.IsiOS11OrNewer or something? Or is that something internal again...

Reply:
```
internal static bool IsiOS11OrNewer
{
	get
	{
		if (!s_isiOS11OrNewer.HasValue)
			s_isiOS11OrNewer = UIDevice.CurrentDevice.CheckSystemVersion(11, 0);
		return s_isiOS11OrNewer.Value;
	}
}
```

Not quite sure why though 🤷